### PR TITLE
fixed DECam config file

### PR DESCRIPTION
**Pull request by igorandreoni**
I made these fixes:

- the available filters are ugrizy, not only r
- horizon key was repeated
- slew rate and overheads were incorrect

### CODE REVIEW

File: gwemopt/data/config/DECam.config [hunk 1]
```diff
@@ -1,4 +1,4 @@
-filt r
+filt ugrizy
```
Comment (mcoughlin) [inline gwemopt/data/config/DECam.config:1]: We use one filter to just scale limiting magnitudes

**Review by mcoughlin — CHANGES_REQUESTED** [review]
Just leave it as r, otherwise good. We would need to build in more capabilities to have multiple filter, magnitude and exposure time keys.

File: gwemopt/data/config/DECam.config (hunk 1 above)
Comment (igorandreoni) [reply gwemopt/data/config/DECam.config]: done

**Comment by mcoughlin** [comment]
@igorandreoni Looks like the test output files need to be updated with this. https://github.com/skyportal/gwemopt/blob/main/gwemopt/tests/test_schedule.py#L62
@robertdstein had left an example here.

**Review by mcoughlin — APPROVED** [review]
Merging. Tests confuse me a bit.

**Pull request merged**